model/src/main/java/org/projectnessie/api/http/HttpTreeApi.java

```diff
@@ -257,6 +257,79 @@ EntriesResponse getEntries(
       @NotNull @Valid @BeanParam EntriesParams params)
       throws NessieNotFoundException;
 
+  /**
+   * Retrieve objects for a ref, potentially truncated by the backend.
```

Comment:
The doc doesn't seem to match the functionality

Reply:
i didn't do a lot of docs, wanted an opinion on the concept first. Fixed up now.